src/test/server/server_test_runner.cpp

```diff
@@ -12,7 +13,7 @@
 
 namespace opossum {
 
-class /* #1357 */ DISABLED_ServerTestRunner : public BaseTest {
+class ServerTestRunner : public BaseTest {
```

Comment:
Are you testing named portals anywhere?

Reply:
Portals are used for operating with cursors. Does Hyrise already support this?
[Here](https://www.postgresql.org/docs/10/protocol-overview.html#PROTOCOL-QUERY-CONCEPTS) is some documentation for this and [here](https://stackoverflow.com/questions/52479293/postgresql-refcursor-and-portal-name) you can find an example. Documentation for this is not that good.

Reply:
If they are untested and not unsupported, what is the advantage of having them in the code?

Reply:
I took this code from the old server implementation. Since there is no support for cursors removing all portal related code should be the way to go.

Reply:
Weird. Can we add that to the documentation and the tests?